api/controller/blog.js

```diff
@@ -6,7 +8,46 @@ exports.getBlog = function(req,res) {
 	Blog.findById(req.params.blog_id,function(err,blog) {
 		if (err)
 			res.send(err);
-		
+
 		res.json(blog);
 	});
-};
\ No newline at end of file
+};
+
+exports.deleteBlog = function(req, res) {
+	Blog.findById(req.params.blog_id, function(err, blog) {
+		if (err) {
+			res.status(500).json(err);
+			return;
+		}
+
+		if (blog != undefined) {
+			apiRoute.AuthChecker(req, res, function() {
```

Comment:
@iit2009060 
When we use custom calls to the endpoint with the authorization header set, the req.user variable is not set.
Since there is no interface right now for logging in and deleting a blog, therefore we have to make custom calls to the endpoint using an app like Postman and send DELETE requests to /blogs/{id}.

Right now Req.user is only set if the user logs in using http://localhost:3000/login or by using AuthChecker